graphql_api/mutation/embedding.py

```diff
@@ -35,8 +52,22 @@ def mutate(self, info, project_id: str, attribute_id: str, embedding_handle: str
         auth.check_demo_access(info)
         auth.check_project_access(info, project_id)
         user = get_user_by_info(info)
-        manager.create_token_level_embedding(
-            project_id, user.id, attribute_id, embedding_handle
+        type = "token"
```

Comment:
enum for the embedding type, also here

Reply:
And I would prefer to call the variable `embedding_type` :)

Reply:
done